user_model.ml

```diff
@@ -0,0 +1,82 @@
+module Rng = Mirage_crypto_rng
+
+type token = {
+  token_type : string;
+  access_token : string;
+  expires_in : int; (* After 1 hour (3660 seconds) of inactivity *)
+}
+let clean_string s =
+  (* Remove backslashes and double quotes from the string *)
+  let buffer = Buffer.create (String.length s) in
+  String.iter (fun c ->
+    match c with
+    | '\\' -> ()
+    | '"' -> ()
+    | _ -> Buffer.add_char buffer c
+  ) s;
+  Buffer.contents buffer
+
+
+let token_to_json t : Yojson.Basic.t  =
+  `Assoc [
+    ("token_type", `String t.token_type);
+    ("access_token", `String t.access_token);
+    ("expires_in", `Int t.expires_in)
+  ]
+
+type user = {
+  name : string;
+  email : string;
+  password : string;
+  uuid : string;
+  token : token option;
```

Comment:
```suggestion
  token : token list;
```

at least in my mind, this will need to be a list (or later maybe a map from token to (expiry, permissions)).